### ISSUE TITLE
PDF Preview button not working

### ISSUE THREAD

**Issue by SinTan1729** [James-Yu/LaTeX-Workshop#3326]
### Disable all the other extensions except for LaTeX Workshop, restart VS Code, and check that you still see this issue. [Required]

You still see this issue?: Yes

### Make sure to visit the [wiki FAQ](https://github.com/James-Yu/LaTeX-Workshop/wiki/FAQ) before filling an issue.

You visited the wiki?: Yes

### If your issue is with compiling a document (not having to do with finding the root file of a project), check first that you can compile manually.

You can compile a TeX document manually?: Yes

## Describe the bug [Required]

The "View LaTeX PDF file" button is not working. I can still open the PDF using the palette or using <kbd>Ctrl+Alt+V</kbd>.

### To Reproduce

1. Open any LaTex project
2. Click on "View LaTeX PDF file" button
3. Nothing happens

### Expected behavior

A tab with the PDF should open.

## Logs [Required]

Please paste the whole log messages here, not parts of ones. The log should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

### LaTeX Workshop Output [Required]

```
[02:40:16] Initializing LaTeX Workshop.
[02:40:16] Extension root: /home/sintan/.vscode/extensions/james-yu.latex-workshop-8.27.1
[02:40:16] $PATH: /home/sintan/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/bin:/home/sintan/.dotnet/tools:/usr/lib/jvm/default/bin:/usr/bin/site_perl:/usr/bin/vendor_perl:/usr/bin/core_perl
[02:40:16] $SHELL: /bin/bash
[02:40:16] $LANG: en_IN.UTF-8
[02:40:16] $LC_ALL: undefined
[02:40:16] process.platform: linux
[02:40:16] process.arch: x64
[02:40:16] vscode.env.appName: Visual Studio Code
[02:40:16] vscode.env.remoteName: undefined
[02:40:16] vscode.env.uiKind: 1
[02:40:16] Workspace for configuration: file:///home/sintan/Documents/Github Projects/CV
[02:40:16] editor.acceptSuggestionOnEnter: "on"
[02:40:16] latex-workshop.bind.enter.key: true
[02:40:16] latex-workshop.docker.enabled: false
[02:40:16] latex-workshop.docker.image.latex: ""
[02:40:16] latex-workshop.hover.preview.mathjax.extensions: []
[02:40:16] latex-workshop.intellisense.package.enabled: true
[02:40:16] latex-workshop.intellisense.update.aggressive.enabled: false
[02:40:16] latex-workshop.intellisense.update.delay: 1000
[02:40:16] latex-workshop.latex.autoBuild.run: "onFileChange"
[02:40:16] latex-workshop.latex.build.forceRecipeUsage: true
[02:40:16] latex-workshop.latex.outDir: "%DIR%"
[02:40:16] latex-workshop.latex.recipes: [
 {
  "name": "latexmk 🔃",
  "tools": [
   "latexmk"
  ]
 },
 {
  "name": "latexmk (latexmkrc)",
  "tools": [
   "latexmk_rconly"
  ]
 },
 {
  "name": "latexmk (lualatex)",
  "tools": [
   "lualatexmk"
  ]
 },
 {
  "name": "latexmk (xelatex)",
  "tools": [
   "xelatexmk"
  ]
 },
 {
  "name": "pdflatex ➞ bibtex ➞ pdflatex × 2",
  "tools": [
   "pdflatex",
   "bibtex",
   "pdflatex",
   "pdflatex"
  ]
 },
 {
  "name": "Compile Rnw files",
  "tools": [
   "rnw2tex",
   "latexmk"
  ]
 },
 {
  "name": "Compile Jnw files",
  "tools": [
   "jnw2tex",
   "latexmk"
  ]
 },
 {
  "name": "tectonic",
  "tools": [
   "tectonic"
  ]
 }
]
[02:40:16] latex-workshop.latex.tools: [
 {
  "name": "latexmk",
  "command": "latexmk",
  "args": [
   "-synctex=1",
   "-interaction=nonstopmode",
   "-file-line-error",
   "-pdf",
   "-outdir=%OUTDIR%",
   "%DOC%"
  ],
  "env": {}
 },
 {
  "name": "lualatexmk",
  "command": "latexmk",
  "args": [
   "-synctex=1",
   "-interaction=nonstopmode",
   "-file-line-error",
   "-lualatex",
   "-outdir=%OUTDIR%",
   "%DOC%"
  ],
  "env": {}
 },
 {
  "name": "xelatexmk",
  "command": "latexmk",
  "args": [
   "-synctex=1",
   "-interaction=nonstopmode",
   "-file-line-error",
   "-xelatex",
   "-outdir=%OUTDIR%",
   "%DOC%"
  ],
  "env": {}
 },
 {
  "name": "latexmk_rconly",
  "command": "latexmk",
  "args": [
   "%DOC%"
  ],
  "env": {}
 },
 {
  "name": "pdflatex",
  "command": "pdflatex",
  "args": [
   "-synctex=1",
   "-interaction=nonstopmode",
   "-file-line-error",
   "%DOC%"
  ],
  "env": {}
 },
 {
  "name": "bibtex",
  "command": "bibtex",
  "args": [
   "%DOCFILE%"
  ],
  "env": {}
 },
 {
  "name": "rnw2tex",
  "command": "Rscript",
  "args": [
   "-e",
   "knitr::opts_knit$set(concordance = TRUE); knitr::knit('%DOCFILE_EXT%')"
  ],
  "env": {}
 },
 {
  "name": "jnw2tex",
  "command": "julia",
  "args": [
   "-e",
   "using Weave; weave(\"%DOC_EXT%\", doctype=\"tex\")"
  ],
  "env": {}
 },
 {
  "name": "jnw2texmintex",
  "command": "julia",
  "args": [
   "-e",
   "using Weave; weave(\"%DOC_EXT%\", doctype=\"texminted\")"
  ],
  "env": {}
 },
 {
  "name": "tectonic",
  "command": "tectonic",
  "args": [
   "--synctex",
   "--keep-logs",
   "%DOC%.tex"
  ],
  "env": {}
 }
]
[02:40:16] latex-workshop.viewer.pdf.internal.keyboardEvent: "auto"
[02:40:16] Creating a new file watcher.
[02:40:16] watcherOptions: {"useFsEvents":false,"usePolling":false,"interval":300,"binaryInterval":1000,"awaitWriteFinish":{"stabilityThreshold":250}}
[02:40:16] Creating PDF file watcher.
[02:40:16] watcherOptions: {"useFsEvents":false,"usePolling":false,"interval":300,"binaryInterval":1000,"awaitWriteFinish":{}}
[02:40:16] Creating Bib file watcher.
[02:40:16] watcherOptions: {"useFsEvents":false,"usePolling":false,"interval":300,"binaryInterval":1000,"awaitWriteFinish":{"stabilityThreshold":250}}
[02:40:16] Set $LATEXWORKSHOP_DOCKER_LATEX: ""
[02:40:16] [Server] Creating LaTeX Workshop http and websocket server.
[02:40:16] LaTeX Workshop initialized.
[02:40:16] Trigger characters for intellisense of LaTeX documents: ["\\",",","{"]
[02:40:16] Bibtex format config: {"tab":"\t","case":"lowercase","left":"{","right":"}","trailingComma":false,"sort":["key"],"alignOnEqual":true,"sortFields":true,"fieldsOrder":[],"firstEntries":["string","xdata"]}
[02:40:16] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:40:16] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:16] Root file changed: from undefined to /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:16] Start to find all dependencies.
[02:40:16] Root file languageId: latex
[02:40:16] [Server] Server successfully started: {"address":"127.0.0.1","family":"IPv4","port":37781}
[02:40:16] Reset file watcher.
[02:40:16] Parsing a file and its subfiles: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:16] Parse fls file.
[02:40:16] Fls file found: /home/sintan/Documents/Github Projects/CV/CV.fls
[02:40:16] Parse aux file: /home/sintan/Documents/Github Projects/CV/CV.aux
[02:40:16] Added to file watcher: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:16] Snippet data loaded.
[02:40:16] [Server] valdOrigin is http://127.0.0.1:37781
[02:40:16] Restoring the PDF viewer at the column 1 from the state: {"type":"state","state":{"pdfFileUri":"file:///home/sintan/Documents/Github Projects/CV/CV.pdf","scale":"auto","scrollMode":0,"spreadMode":0,"scrollTop":0,"scrollLeft":0,"trim":0,"synctexEnabled":true,"autoReloadEnabled":true}}
[02:40:16] The internal PDF viewer url: http://127.0.0.1:37781/viewer.html?file=pdf..ZmlsZSUzQSUyRiUyRiUyRmhvbWUlMkZzaW50YW4lMkZEb2N1bWVudHMlMkZHaXRodWIlMjBQcm9qZWN0cyUyRkNWJTJGQ1YucGRm
[02:40:16] Added to PDF file watcher: file:///home/sintan/Documents/Github Projects/CV/CV.pdf
[02:40:17] Checking for duplicate labels: /home/sintan/Documents/Github Projects/CV/CV.tex.
[02:40:17] Handle data type: open
[02:40:17] Preview PDF file: file:///home/sintan/Documents/Github Projects/CV/CV.pdf
[02:40:17] Handle data type: loaded
[02:40:18] Manager.fileWatcher.getWatched: {"/home/sintan/Documents/Github Projects/CV":["CV.tex"]}
[02:40:18] Manager.filesWatched: ["/home/sintan/Documents/Github Projects/CV/CV.tex"]
[02:40:18] BibWatcher.bibWatcher.getWatched: {}
[02:40:18] BibWatcher.bibsWatched: []
[02:40:18] PdfWatcher.pdfWatcher.getWatched: {"/home/sintan/Documents/Github Projects/CV":["CV.pdf"]}
[02:40:18] PdfWatcher.pdfsWatched: ["/home/sintan/Documents/Github Projects/CV/CV.pdf"]
[02:40:18] PdfWatcher.watchedPdfVirtualUris: []
[02:40:18] PdfWatcher.ignoredPdfUris: []
[02:40:21] VIEW command invoked with mode: file:///home/sintan/Documents/Github%20Projects/CV/CV.tex.
[02:40:21] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:40:21] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:21] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:21] VIEW command invoked with mode: file:///home/sintan/Documents/Github%20Projects/CV/CV.tex.
[02:40:21] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:40:21] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:21] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:21] VIEW command invoked with mode: file:///home/sintan/Documents/Github%20Projects/CV/CV.tex.
[02:40:21] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:40:21] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:21] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:24] VIEW command invoked with mode: file:///home/sintan/Documents/Github%20Projects/CV/CV.tex.
[02:40:24] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:40:24] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:24] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:25] VIEW command invoked with mode: file:///home/sintan/Documents/Github%20Projects/CV/CV.tex.
[02:40:25] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:40:25] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:25] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:25] VIEW command invoked with mode: file:///home/sintan/Documents/Github%20Projects/CV/CV.tex.
[02:40:25] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:40:25] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:25] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:25] VIEW command invoked with mode: file:///home/sintan/Documents/Github%20Projects/CV/CV.tex.
[02:40:25] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:40:25] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:25] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:27] VIEW command invoked.
[02:40:27] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:40:27] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:27] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:27] The internal PDF viewer url: http://127.0.0.1:37781/viewer.html?file=pdf..ZmlsZSUzQSUyRiUyRiUyRmhvbWUlMkZzaW50YW4lMkZEb2N1bWVudHMlMkZHaXRodWIlMjBQcm9qZWN0cyUyRkNWJTJGQ1YucGRm
[02:40:27] Open PDF tab for file:///home/sintan/Documents/Github Projects/CV/CV.pdf
[02:40:27] Handle data type: open
[02:40:27] Preview PDF file: file:///home/sintan/Documents/Github Projects/CV/CV.pdf
[02:40:27] Handle data type: loaded
[02:40:28] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:40:28] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:40:28] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:28] VIEW command invoked.
[02:43:28] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:43:28] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:28] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:28] The internal PDF viewer url: http://127.0.0.1:37781/viewer.html?file=pdf..ZmlsZSUzQSUyRiUyRiUyRmhvbWUlMkZzaW50YW4lMkZEb2N1bWVudHMlMkZHaXRodWIlMjBQcm9qZWN0cyUyRkNWJTJGQ1YucGRm
[02:43:28] Open PDF tab for file:///home/sintan/Documents/Github Projects/CV/CV.pdf
[02:43:29] Handle data type: open
[02:43:29] Preview PDF file: file:///home/sintan/Documents/Github Projects/CV/CV.pdf
[02:43:29] Handle data type: loaded
[02:43:29] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:43:29] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:29] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:37] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:43:37] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:43:37] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:37] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:37] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:37] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:39] VIEW command invoked.
[02:43:39] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:43:39] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:39] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:39] The internal PDF viewer url: http://127.0.0.1:37781/viewer.html?file=pdf..ZmlsZSUzQSUyRiUyRiUyRmhvbWUlMkZzaW50YW4lMkZEb2N1bWVudHMlMkZHaXRodWIlMjBQcm9qZWN0cyUyRkNWJTJGQ1YucGRm
[02:43:40] Open PDF tab for file:///home/sintan/Documents/Github Projects/CV/CV.pdf
[02:43:40] Handle data type: open
[02:43:40] Preview PDF file: file:///home/sintan/Documents/Github Projects/CV/CV.pdf
[02:43:40] Handle data type: loaded
[02:43:41] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:43:41] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:41] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:43] VIEW command invoked with mode: file:///home/sintan/Documents/Github%20Projects/CV/CV.tex.
[02:43:43] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:43:43] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:43] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:43] VIEW command invoked with mode: file:///home/sintan/Documents/Github%20Projects/CV/CV.tex.
[02:43:43] Current workspace folders: ["file:///home/sintan/Documents/Github Projects/CV"]
[02:43:43] Found root file from active editor: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:43:43] Keep using the same root file: /home/sintan/Documents/Github Projects/CV/CV.tex
[02:45:37] LOG command invoked: default
[02:45:43] LOG command invoked: default
```
[Link to logs.](https://pastebin.com/U218cndS)

### Developer Tools Console [Required]

```
To access the log, click 'help' -> 'Toggle Developer Tools' -> 'Console'. Paste anything suspicious.

[Paste the log here]
```

## Desktop [Required]

Please write exact version numbers. Please don't write `latest` instead of exact numbers.

 - OS: Linux
 - VS Code version: 1.67.2
 - Extension version: 8.27.1
 - TeX distribution version: TeX Live 2022/Arch Linux

### Are you using VSCodium?

No

### Are you using the Snap or Flatpack versions of VS Code?

No

### Are you using LaTeX Workshop with VS Code Remote?

No

If the answer is Yes, please write which one you are using. Write the versions of the remote extension.

- Remote Containers/Remote SSH/Remote WSL
- Remote Extension Version: [e.g. 0.48.0]

**Comment by gaofengnan** [comment]
The same here.  I had to revert to the 8.26.0 version to get it working again.

**Comment by aywi** [comment]
Yes, the same on Windows 10. The latest fix https://github.com/James-Yu/LaTeX-Workshop/commit/58dae37c3e8550c03cb196268c299eb3bfe19d07 doesn't seem to work.

**Comment by tamuratak** [comment]
@James-Yu the released build v8.27.1 is not the latest fixed version.

**Comment by tamuratak** [comment]
Fixed with 3aad09b3419b3a8b077c6d4ee5b6f49a9cc19b46
